src/pynxtools/dataconverter/readers/multi/reader.py

```diff
@@ -163,10 +195,11 @@ def try_convert(value: str) -> Union[str, float, int, bool]:
 
     if prefixes and new_entry_dict[key] is None:
         del new_entry_dict[key]
-        logger.warning(
-            f"Could not find value for key {key} with value {value}.\n"
-            f"Tried prefixes: {prefixes}."
-        )
+        if not suppress_warning:
+            logger.warning(
```

Comment:
Consider setting a level for the warnings and adapt the `logger.level` appropriately. That way you don't have to pass the `suppress_warning` through every function call. However, this is not to critical and I think it's also fine to merge this with the `suppress_warning`

Reply:
Yeah, I thought about that, but the idea was that the reader that inherits MultiFormatReader can set that level itself. Wasn't sure how to pass the logger.level adjustments properly through the functions. For the XpsReader, I just want this particular call suppressed, while for the rest of the function, the level shall be at logging.INFO.